src/lib/docs.ts

```diff
@@ -0,0 +1,27 @@
+import path from 'path'
+import { logInfoMessage, logError } from './logger.js'
+import Generator from '@asyncapi/generator'
+
+export default async (spec, config, resDir) => {
+  const configData = config.docs
+  if (!configData || configData?.enabled) {
```

Comment:
Something like this @fmvilas

Reply:
What about the case when `configData` exists but `enabled` is not defined? It should still work but with this check it won't. The reason I said to use `disabled` instead is that you can easily check for it like this:

```js
  if (configData?.disabled) return
```

This way, you early return **only** when you find that `disabled` is true.

Reply:
Alternatively, you can check for this:

```js
  if (configData?.enabled === false) return
```

That should work as well, I think.

Reply:
But seems more logical to me that, since the default is that it generates docs, you only go there when you want to disable it or when you want to change the template.

Reply:
Ah! Now I understand your point @fmvilas

Reply:
Updated the PR @fmvilas. wyt?